README.md

```diff
@@ -15,6 +15,15 @@ For community based support with deCONZ or Phoscon, please visit the [deCONZ Dis
 The Phoscon App is a browser based web application and supports lights, sensors and switches. For more information and screenshots visit the [Phoscon App Documentation](https://phoscon.de/app/doc?ref=gh).
 
 
+### Release Schedule
+
+deCONZ is having a beta release at the 15th day of the month in which the beta of last month becomes stable. Pull requests done before the 10th of the month, are getting included in the next beta (if there are no issues ofcourse!).  Stable release would be around the 1st - 5th of the month after latest beta. 
+Current Beta: 2.05.82
+Current Stable: 2.05.81
+
+Next Beta: 2.05.84 expected at the 15th of October. 
+Next Stable: 2.05.83 expected in week 45 (2th to 8th of November)
```

Comment:
Due the hot fix we are already on beta 2.05.83.

* Next Stable: 2.05.84 available until 8th October (minor fixes and new firmware files)
* Next Beta: 2.05.85 15th of October

Reply:
Edited!